after/syntax/c.vim

```diff
@@ -233,9 +233,9 @@ syn keyword	cAnsiName	STDC FENV_ACCESS
 syn keyword	cAnsiName	STDC FP_CONTRACT
 
 syn keyword cAnsiName AF_INET SOCK_STREAM INADDR_ANY AF_INET
-235 syn keyword cAnsiName SOL_SOCKET SO_REUSEPORT SO_REUSEADDR
-236 syn keyword cAnsiName SO_RCVTIMEO IPPROTO_TCP TCP_NODELAY
-237 syn keyword cAnsiName SOCK_DGRAM POLLIN
+syn keyword cAnsiName SOL_SOCKET SO_REUSEPORT SO_REUSEADDR
+syn keyword cAnsiName SO_RCVTIMEO IPPROTO_TCP TCP_NODELAY
+syn keyword cAnsiName SOCK_DGRAM POLLIN
```

Comment:
Are these really ANSI C, or POSIX?

Not sure it's a good idea to start adding "common" things instead of just "ansi" things.

Reply:
They are POSIX :-(

Reply:
Do you have group cPosixName? I could add to that instead.